serde/src/test/kotlin/com/amazon/ionhiveserde/Util.kt

```diff
@@ -15,7 +15,53 @@
 
 package com.amazon.ionhiveserde
 
+import com.amazon.ion.IonDatagram
+import com.amazon.ion.IonSequence
+import com.amazon.ion.IonStruct
 import com.amazon.ion.system.IonSystemBuilder
+import com.amazon.ionhiveserde.caseinsensitivedecorator.IonSequenceCaseInsensitiveDecorator
+import com.amazon.ionhiveserde.caseinsensitivedecorator.IonStructCaseInsensitiveDecorator
 
 internal val ION = IonSystemBuilder.standard().build()
 internal val ionNull = ION.newNull()
+
+internal fun datagram_for(s: String): IonDatagram {
+    return ION.loader.load(s)
+}
+
+internal fun struct_for(s: String): IonStruct {
+    val v = datagram_for(s).iterator().next()
+    if (v !is IonStruct) throw IllegalArgumentException("Required an IonStruct, found ${v.javaClass.simpleName}")
+
+    return v
+}
+
+internal fun case_insensitive_decorator_struct_for(s : String): IonStruct {
+    return IonStructCaseInsensitiveDecorator(struct_for(s))
+}
+
+internal fun sequence_for(s: String): IonSequence {
+    val v = datagram_for(s).iterator().next()
+    if (v !is IonSequence) throw IllegalArgumentException("Required an IonSequence, found ${v.javaClass.simpleName}")
+
+    return v
+}
+
+internal fun case_insensitive_decorator_sequence_for(s : String): IonSequence {
+    return IonSequenceCaseInsensitiveDecorator(sequence_for(s))
+}
+
+/**
+ * Create a sample struct used for testing. To create a desired Ion struct, use struct_for().
+ */
+internal fun makeStruct(): IonStruct {
+    return struct_for("{a: 1, b: 2, c: 3}")
+}
```

Comment:
We ought to inline this method. Helper methods are to reduce work for the reader and increase clarity- burying the details of the constructed struct inside this helper obscures but without a payoff to make it worth it. The direct `struct_for` invocation will do just fine.

Reply:
Got it, changed to inline for better read experience.